File: app/src/main/java/com/minwook/mytriphistory/model/ContentsModel.java
package com.minwook.mytriphistory.model;

import android.content.Context;

import com.minwook.mytriphistory.DAO.Content;
import com.minwook.mytriphistory.common.DumyData;

import java.util.ArrayList;

public class ContentsModel {

    private Context context;
    private ArrayList<Content> list;

    public ContentsModel(Context context){
        this.context = context;
    }

    public ArrayList<Content> dataLoad(){
        return DumyData.getInstance().getList();
    }
}

File: app/src/main/java/com/minwook/mytriphistory/kotlin/model/ContentsModel.kt
package com.minwook.mytriphistory.kotlin.model

import android.content.Context
import com.minwook.mytriphistory.kotlin.DAO.Content
import com.minwook.mytriphistory.kotlin.common.DumyData

class ContentsModel {


    fun dataLoad(): ArrayList<Content> {
        return DumyData.list
    }
}